idb/postgres/postgres_migrations.go

```diff
@@ -35,6 +36,8 @@ func init() {
 		{m4RewardsAndDatesPart1, true, "Update DB Schema for cumulative account reward support and creation dates."},
 		{m5RewardsAndDatesPart2, false, "Compute cumulative account rewards for all accounts."},
 		{m6MarkTxnJSONSplit, true, "record round at which txn json recording changes, for future migration to fixup prior records"},
+		{m7StaleClosedAccounts, false, "clear some stale data from closed accounts"},
+		{m8TxnJsonEncoding, false, "some txn JSON encodings need app keys base64 encoded"},
```

Comment:
m5 has not been released, it's a long blocking migration so it needs to happen at the end

Reply:
m7 is trivially small. 3 lines of sql.
m8 is structurally different, it needs to scan every txn and maybe update a field. (it's based on the copy of the prior txid update)

Reply:
m6 could be marked non-blocking. It doesn't actually need to be precise. The json update is harmless if it reads a few more txns and find the json already in good form because it was written after the bugfix started running.

Reply:
How long will it take to run all of these? it seems like m8 is as long as m5.